src/karmen_frontend/src/components/gateways/heartbeat.js

```diff
@@ -46,24 +46,22 @@ const UpgradeModal = ({ shouldShow }) => {
     isOpen && (
       <Modal>
         <h1 className="modal-title text-center">
-          Application has been upgraded
+          Application has been upgraded, please reload the page to continue.
         </h1>
-        <p className="message-success">
-          There is a new version of the application. Upgrade now, please.
-        </p>
-        <p className="text-center">
-          If the button does not work, try pressing <code>Ctrl+R</code> or{" "}
-          <code>&#8984;+R</code>.
-        </p>
         <div className="cta-box text-center">
           <button
             className="btn"
             onClick={() => {
               window.location.reload(true);
             }}
           >
-            Upgrade
+            Reload
           </button>
+
+          <small className="text-center">
+            If the button does not work,<br/> try pressing <code>Ctrl+R</code> or{" "}
```

Comment:
I do not understand this statement. How can it not work? This doesn't look like clean UX.

Reply:
For example, whole application can somehowe become unresponsive.

Reply:
OK, let's leave it this way.